src/components/layout/Panel.tsx

```diff
@@ -98,9 +98,11 @@ const PanelNext = function Panel({
         <CardTitle>{title}</CardTitle>
       </CardHeader>
       {scrollable ? <Scroll>{panelContent}</Scroll> : <>{panelContent}</>}
-      <CardContent>
-        {buttons && <CardActions>{buttons}</CardActions>}
-      </CardContent>
+      {buttons && (
+        <CardContent data-testid="panel-buttons">
+          {buttons && <CardActions>{buttons}</CardActions>}
```

Comment:
You probably don't need to check the buttons here again.

```suggestion
          <CardActions>{buttons}</CardActions>
```

Reply:
🤦🏻‍♀️